eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/AbstractMutableList.java

```diff
@@ -1204,7 +1204,7 @@ public int binarySearch(T key)
                 MutableList<T> batch = new FastList<>(Math.min(size, this.size() - i));
 
                 for (int j = 0; j < size && i < this.size(); j++)
-                {
+            {
```

Comment:
Thanks for making the changes!

Actually this remaining change is breaking the format of curly braces. It would be great if you could squash the 2 commits into one and also completely revert the changes in this file.